sdk/js/src/cosmos/address.ts

```diff
@@ -0,0 +1,8 @@
+import { bech32 } from "bech32";
+
+export function canonicalAddress(humanAddress: string) {
+  return new Uint8Array(bech32.fromWords(bech32.decode(humanAddress).words));
+}
+export function humanAddress(hrp: string, canonicalAddress: Uint8Array) {
```

Comment:
Just thinking out loud here, and you have probably already thought about it, so you can feel free to just close this issue. 

This is a breaking change (adding a parameter). Would it be better to leave this function as is and create a separate one that takes the extra parameter?

Reply:
It's been decided to break backwards compatibility and add the new parameter to the function since it's no longer terra specific